ref/hash_sha256.c

```diff
@@ -35,7 +54,7 @@ void prf_addr(unsigned char *out, const unsigned char *key,
 /**
  * Computes the message-dependent randomness R, using a secret seed as a key
  * for HMAC, and an optional randomization value prefixed to the message.
- * This requires m to have at least SPX_SHA256_BLOCK_BYTES + SPX_N space
+ * This requires m to have at least SPX_SHAX_BLOCK_BYTES + SPX_N space
```

Comment:
This comment is out of date — now we're updating it we can correct it.  There is no randomisation value prefixed to the message nor a buffer.